checks/evaluation/signed_releases.go

```diff
@@ -37,6 +38,7 @@ func SignedReleases(name string,
 	expectedProbes := []string{
 		releasesAreSigned.Probe,
 		releasesHaveProvenance.Probe,
+		releasesHaveVerifiedProvenance.Probe,
```

Comment:
if we just skip over the results, let's not expect it to be in the eval code yet